apps/greeter/web/router.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Greeter.Router do
+  use Greeter.Web, :router
+
+  pipeline :browser do
+    plug :accepts, ["html"]
+    plug :fetch_session
+    plug :fetch_flash
+    plug :protect_from_forgery
+    plug :put_secure_browser_headers
+  end
+
+  pipeline :api do
+    plug :accepts, ["json"]
+  end
+
+  scope "/", Greeter do
+    pipe_through :browser # Use the default browser stack
+
+    get "/", PageController, :index
+  end
+
+  # Test conflictive routes
+  get "/starwars", Greeter.PageController, :index
+  get "/star-wars", Greeter.PageController, :index
+  get "/starwars/star-wars", Greeter.PageController, :index
+
```

Comment:
I think it depends on the Greeter.Router module:

https://github.com/iporaitech/pwr2-docker/pull/30/files/3ad85a018059350f2b5ef9cb5920a776c23d1254#diff-c48864976f0498395ce1084ef2960196R25

But this normally isn't a problem, I think the problem have something to do with the way those modules are loaded in an umbrella project or with the Endpoint